app/src/main/java/ch/epfl/sdp/drone3d/ui/mission/MappingMissionSelectionActivity.kt

```diff
@@ -54,9 +54,9 @@ class MappingMissionSelectionActivity : AppCompatActivity() {
     private fun setupAdapter(
             data: LiveData<List<MappingMission>>,
             adapter: ListAdapter<MappingMission, out RecyclerView.ViewHolder>
-    ) = data.observe(this) {
+    ) = data.observe(this, androidx.lifecycle.Observer {
```

Comment:
Why refractor this code ?

Reply:
On my computer, everything was underlined in red with the older version, so I refactored it so that Android studio doesn't complain anymore